### PR TITLE
fixes #211 : Adds an export button to save conversation

### LINKED ISSUE #211: Sharing Conversations on Social-Media or export
**I'm submitting a ...**  (check one with "x")
- [ ] bug report
- [x] feature request
- [ ] support request => Please do not submit support requests here, instead ask your query in out Gitter channel at https://gitter.im/fossasia/susi_server

**Current behavior:**
Conversations with an ai bot can sometimes become very interesting and sending screenshots to people can be tedious often. To share a conversation, it's really tough.

**Expected behavior:**
A simple button will allow the user to choose which option he/she wants.
- Export transcript of chat in ( `json`/`xml`)
***or***
- Get a big image `(.jpg`/ or anything) of the chat
- PDF of the chat in readable format

**Steps to reproduce:**
- saving the conversation to a variable (array).
- adding buttons to interface with function bindings
- prompt to download 

**Other information:**

**Would you like to work on the issue?**

For now, i'm researching the issue. Can't assure my contribution for now.

### PR DESCRIPTION
**Pull request by YashKumarVerma**
Fixes #211 

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `master` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

#### Changes proposed in this pull request:

-added export option

### CODE REVIEW

**Comment by YashKumarVerma** [comment]
Can anyone help me solving that error. I did nothing with that fragment of code

**Comment by ms10398** [comment]
I think shifting the declaration line up will do it or removing the declaration and where its inintialised declare it there it will work

**Comment by YashKumarVerma** [comment]
@gabru-md @ms10398 @stealthanthrax Everything done 😄

File: src/popup.html [hunk 1]
```diff
@@ -33,11 +33,12 @@
                 <a href="https://chat.susi.ai/overview" target="_blank"><li>About<img src="images/about.svg" ></li></a>
                 <a href="https://chat.susi.ai/" target="_blank"><li>Chat<img src="images/chat.svg"></li></a>
                 <a href="https://skills.susi.ai/" target="_blank"><li>Skills<img src="images/skills.svg"></li></a>
+                <a id="export"><li>Export<img src="images/export.svg"></li></a>
                 <a href="" id="clear"><li>Clear Chat<img src="images/clearAll.svg" ></li></a>
                 <a id="settings" target="_blank"><li>Settings<img src="images/settings.svg" ></li></a>
                 <a href="login.html" target="_blank"><li>Login<img src="images/login.svg" > </li></a>
             </div>
-        </div>
+        </div>
```
Comment (0x48piraj) [inline src/popup.html]: Why this ?

**Comment by YashKumarVerma** [comment]
@0x48piraj 
changed done 👍

**Comment by YashKumarVerma** [comment]
screenshot 283
FINALLY DONE 🔥

**Pull request merged**